doc/users/installing/index.rst

```diff
@@ -1,9 +1,21 @@
 .. redirect-from:: /users/installing
 
-============
+************
 Installation
+************
+
+Dependencies
 ============
 
+Mandatory dependencies should be installed automatically if you install Matplotlib using
+a package manager such as ``pip`` or ``conda``; therefore this list is primarily for
+reference and troubleshooting.
```

Comment:
```suggestion
Mandatory dependencies should be installed automatically if you install Matplotlib using
a package manager.
```

Given that for most users this is automatically handled, maybe move this section to the end?

Reply:
We should not lead with content that most users do not need to know.